### PR TITLE
sunday night changes

### PR DESCRIPTION
**Pull request by andreazzabr**
changes of hard coded things into variables.
more to come tomorrow

### CODE REVIEW

File: terraform/rds/main.tf [hunk 1]
```diff
@@ -18,7 +18,7 @@ resource "aws_rds_cluster_instance" "cluster_instances" {
     count                  = 1
     identifier             = "${var.project_name}-rds-instance"
     cluster_identifier     = aws_rds_cluster.rdsclu.id
-    instance_class         = "db.r4.large" # this is the smallest instance can be use for the engine
+    instance_class         = "db.t2.small" # this is the smallest instance can be use for the engine
```
Comment (hsy3418) [inline terraform/rds/main.tf:21]: Have you tested it? I tested it before, it was saying t2.samll is not supported for aurodb.

File: terraform/ecs/variables.tf [hunk 2]
```diff
@@ -16,4 +16,5 @@ variable "db_passowrd_arn" {type = string}
 variable "db_name_arn" {type = string}
 variable "pub_sub1" {type = string}
 variable "pub_sub2" {type = string}
-variable "target_group_arn" {type = string}
\ No newline at end of file
+variable "target_group_arn" {type = string}
+variable containerimage {type=string}
```
Comment (hsy3418) [inline terraform/ecs/variables.tf:20]: is it correct that containerimage don't have quotes on it?

File: terraform/rds/main.tf (hunk 1 above)
Comment (andreazzabr) [reply terraform/rds/main.tf:21]: yep! it works for me

File: terraform/ecs/variables.tf (hunk 2 above)
Comment (andreazzabr) [reply terraform/ecs/variables.tf:20]: yes, all variables on terraform 0.12 or newer don't need ""  or ${}

**Review by hsy3418 — APPROVED** [review]

**Pull request merged**